src/utils/resolveYextEntityField.ts

```diff
@@ -38,9 +37,6 @@ export const resolveYextEntityField = <T>(
     console.error("Error in resolveYextEntityField:", e);
   }
 
-  // if field not found, return constant value as a fallback
-  console.warn(
-    `The field ${entityField.field} was not found in the document, defaulting to constant value ${entityField.constantValue}.`
-  );
-  return entityField.constantValue as T;
+  console.warn(`The field ${entityField.field} was not found in the document.`);
```

Comment:
Maybe have a separate warning for if field === ""? Since that would be because the user didn't select a field rather than the field not existing in the document.

Reply:
oops good catch! don't wanna spam the logs haha

Reply:
Good idea